build_runner/test/common/test_phases.dart

```diff
@@ -144,7 +144,7 @@ void checkBuild(BuildResult result,
 
   final unhiddenOutputs = <String, dynamic>{};
   final unhiddenAssets = new Set<AssetId>();
-  for (final id in outputs?.keys ?? const []) {
+  for (final String id in outputs?.keys ?? const []) {
```

Comment:
nit: we should try to get more consistent about final vs not in for loops... :)

not a blocker anything as its a preexisting issue

Reply:
Yeah I know that the larger community pushed back against "final everywhere" but I find that I occasionally slip back into that habit...

Reply:
ya honestly I don't care, I can see it either way, but I do care that it is consistent :)